src/parser/parser.mly

```diff
@@ -249,6 +249,9 @@ term_:
   | FRESH x=opt_name(ml_name) COLON t=ty_term
     { Sugared.Fresh (x, t) }
 
+  | ABSTRACT c1=prefix_term c2=prefix_term
```

Comment:
I think this is in the wrong place. It should be in the same place as `CONVERT`.